rqt_py_common/CMakeLists.txt

```diff
@@ -22,5 +22,7 @@ install(DIRECTORY resource
 )
 
 if(CATKIN_ENABLE_TESTING)
-  catkin_add_nosetests(test)
+  catkin_add_nosetests(test/test_rqt_common_unit.py)
+  catkin_add_nosetests(test/test_rqt_ros_graph.py)
+  catkin_add_nosetests(test/test_rqt_roscomm_util.py)
```

Comment:
Same as https://github.com/ros-visualization/qt_gui_core/pull/123#discussion_r226803085

Reply:
@dirk-thomas I'm not sure that nosetest is working as expected here. I dug into the catkin_tools code but it became clear that it would lead to serious scope creep. This seems like the most straightforward way to fix the issue and I'm inclined to leave it unless you strongly object

Reply:
The actual problem is that two of the test files have the executable bit set. Nosetest won't pick executables. So removing the executable bit will allow to keep passing the directory name.

Reply:
This seems like it needs to be documented somewhere..

Reply:
This did not fix the issue...

Reply:
@dirk-thomas Can you elaborate on what you mean by 'two files have the executable bit set'? I deleted out the main from the test files and found the same issue.

Reply:
> This seems like it needs to be documented somewhere.

It is documented: https://nose.readthedocs.io/en/latest/usage.html#extended-usage

> It is important to note that the default behavior of nose is to not include tests from files which are executable.